src/components/LbFooter.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div>
+    <div class="app-footer">
+      <div class="row expanded">
+        <div class="columns">
+          <nav>
+            <ul class="inline-list">
+              <li>
+                <popover-link :options="this.popoverLinkOptions"
+                              :slots="this.popoverLinkSlots"
+                              :customStyle="this.customStyle"
+                />
+              </li>
+              <li>
+                <a target="_blank" :href="this.feedbackUrl">Feedback</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+
+<script>
+
+  export default {
+    name: 'LbFooter',
+    components: {
+      PopoverLink: () => import(/* webpackChunkName: "lblb_pvc_PopoverLink" */'@philly/vue-comps/src/components/PopoverLink.vue'),
+    },
+    computed: {
+      feedbackUrl() {
+        if (this.$config.footer) {
+          if (this.$config.footer.feedbackUrl) {
+            return this.$config.footer.feedbackUrl
+          } else {
+            return null
+          }
+        } else {
+          return null
+        }
+      },
+      customStyle() {
+        if (this.$config.footer) {
+          if (this.$config.footer.helpPopover) {
+            if (this.$config.footer.helpPopover.linkStyle) {
+              return this.$config.footer.helpPopover.linkStyle
+            } else {
+              return { 'color': 'white', 'border-bottom': '0px' }
+            }
+          } else {
+            return { 'color': 'white', 'border-bottom': '0px' }
+          }
+        } else {
+          return { 'color': 'white', 'border-bottom': '0px' }
+        }
+      },
+      popoverHeight() {
+        if (this.$config.footer) {
+          if (this.$config.footer.helpPopover) {
+            if (this.$config.footer.helpPopover.height) {
+              return this.$config.footer.helpPopover.height
+            } else {
+              return '80%'
+            }
+          } else {
+            return '80%'
+          }
+        } else {
+          return '80%'
+        }
```

Comment:
I feel like there's a cleaner way to write these conditionals... nested if statements can be _so_ hard to read.

Reply:
It looks like we could rewrite this as: 
```
popoverHeight() {
if (this.$config.footer & this.$config.footer.helpPopover && this.$config.helpPopover.height) {
  return this.$config.footer.helpPopover.height;
}

return 80%;
```
}